base/set.jl

```diff
@@ -191,6 +191,23 @@ function unique(f::Callable, C)
 end
 
 """
+	unique!(itr)
+
+Removes recurrences of items and returns the modified collection
+"""
+function unique!(itr)
+	seen = Set{eltype(itr)}()
```

Comment:
Please use 4 spaces rather than tabs for indentation for consistency with the rest of the repo.